models/models.go

```diff
@@ -187,7 +187,7 @@ type Route struct {
 	Certificate    Certificate
 	UserData       UserData
 	UserDataID     int
-	DefaultTTL     int64
+	DefaultTTL     int64 `gorm:"default:86400"`
```

Comment:
Isn't this going to keep the default as 86400 as it's set to 0? Doc ref http://gorm.io/docs/create.html#Default-Values

Reply:
The default for when it's provisioned is specified here: https://github.com/alphagov/paas-cdn-broker/blob/c86667a7f2a2f99a19f7c9ae75ecdddd164c92aa/config/config.go#L29

This `gorm` default is just for the migration of the existing users - without this, the TTL for the existing tenants will be reset to 0 the first time they run an update

Reply:
Theoretically this can be ripped out once the initial migration is done (after the first deploy)